Babylon/commands/api/scenarioruns/cumulated_logs.py

```diff
@@ -22,18 +23,13 @@ def cumulated_logs(state: Any, azure_token: str, organization_id: str, scenarior
     """
     Get the cumulated logs for the scenarioRun
     """
-    state = state['services']
-    state['api']['organization_id'] = organization_id or state['api']['organization_id']
-    state['api']['scenariorun_id'] = scenariorun_id or state['api'].get('scenariorun_id')
-    if state['api']['scenariorun_id'] is None:
-        logger.error(f"scenariorun : {state['api']['scenariorun_id']} does not exist")
-        return CommandResponse.fail()
-
+    scenariorun_state = state['services']
```

Comment:
elsewhere we use _service_state_, is there a special need to precise _scenariorun_state_ ? :slightly_smiling_face: